internal/contract/types.go

```diff
@@ -151,7 +152,7 @@ func (i *Duration) Get(obj *unstructured.Unstructured) (*metav1.Duration, error)
 	}
 
 	d := &metav1.Duration{}
-	if err := d.UnmarshalJSON([]byte(durationString)); err != nil {
+	if err := d.UnmarshalJSON([]byte((fmt.Sprintf("%q", durationString)))); err != nil {
```

Comment:
```suggestion
	if err := d.UnmarshalJSON([]byte((strconv.Quote(durationString)))); err != nil {
```
nit: let's use strconv.Quote instead